api/tests/integration/AuthTest.php

```diff
@@ -487,10 +453,7 @@ public function testProviderCallbackExistingUser()
 
         $token = str_replace('jwtAuthToken=', '', $tokenParam);
```

Comment:
This should be defined as a constant in the appropriate controller (same as the cookie)